packages/docs/src/routes/docs/menu.md

```diff
@@ -150,7 +150,7 @@
 - [Insights](/docs/labs/insights/index.mdx)
 - [Typed Routes](/docs/labs/typed-routes/index.mdx)
 - [Devtools](/docs/labs/devtools/index.mdx)
-- [usePreventNavigate](/docs/labs/usePreventNavigate/index.mdx)
+- [usePreventNavigate](</docs/labs/usePreventNavigate/index.mdx>)
```

Comment:
@iitzIrFan Do these `<` and `>` work? In any case I don't think they are needed. Could you remove them all please?